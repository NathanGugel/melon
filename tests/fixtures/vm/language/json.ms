
func testJson |file| => {
    let result = json.parse(file);
    io.print(json.stringify(result));
}

func testParse |file| => {
    let result = json.parse(file);
}

testJson("fixtures/vm/language/json_test_files/value.json");
testJson("fixtures/vm/language/json_test_files/array.json");
testParse("fixtures/vm/language/json_test_files/object.json");
testParse("fixtures/vm/language/json_test_files/test1.json");