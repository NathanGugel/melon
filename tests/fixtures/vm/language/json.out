"single value"
["simple array","of","values"]
